Resources/Locale/en-US/job/job-description.ftl

```diff
@@ -36,10 +36,10 @@ job-description-passenger = Enjoy your stay aboard the station with no obligatio
 job-description-psychologist = Provide emotional support to traumatized crew. Currently available on Box, Marathon and Origin Stations.
 job-description-qm = Manage the supplies of the station & the cargo department, keep the salvage specialists working, make sure all orders are fulfilled, and keep the money flowing.
 job-description-rd = Manage the science department, unlocking technologies, acquiring & researching artifacts, and performing experiments.
-job-description-research-assistant = Learn the basics of how to research various artifacts and anomalies.
-job-description-reporter = Entertain & inform the crew with your vibrant journalism through wireless cameras and the radio. Currently available on Bagel Station and Core.
-job-description-salvagespec = Use the salvage magnet to draw in derelicts & asteroids to loot and enrich the station, and fight off any space fauna along the way.
-job-description-scientist = Research alien artifacts, unlock new technologies, upgrade machines around the station, and make everything run more efficiently.
+job-description-research-assistant = Learn the basics of how to research various artifacts, anomalies and robotics.
+job-description-reporter = Entertain & inform the crew with your vibrant journalism through wireless cameras, the radio and the news. Currently available on Bagel Station and Core.
+job-description-salvagespec = Use the salvage magnet to draw in detatched scraps & asteroids to loot and enrich the station, build a salvage ship, and then travel to new planets. and fight off any space fauna along the way.
```

Comment:
Last sentence has incorrect struct, can't start with a lowercase, perhaps replace with "... build a salvage ship and then travel to new planets, while fighting off any space fauna along the way."